models/gamma/gamma_uniswap_pools.sql

```diff
@@ -0,0 +1,40 @@
+ {{
+  config(
+        schema='gamma',
+        alias='uniswap_pools',
+        materialized = 'incremental',
```

Comment:
similar to arrakis, can this be a view?

Reply:
if view, can leave no test in schema. otherwise, please add

Reply:
making a table